tbconnect/serializers.py

```diff
@@ -43,11 +43,11 @@ class Meta:
 
 
 class TBCheckCciDataSerializer(serializers.Serializer):
-    msisdn = PhoneNumberField(required=True)
-    name = serializers.CharField(required=False, allow_blank=True, allow_null=True)
-    language = serializers.CharField(required=True)
-    tb_risk = serializers.CharField(required=True)
-    responded = serializers.CharField(required=True)
-    tb_tested = serializers.CharField(required=True)
-    tb_test_results = serializers.CharField(required=True)
-    screen_timeStamp = serializers.CharField(required=True)
+    CLI = PhoneNumberField(required=True)
+    Name = serializers.CharField(required=False, allow_blank=True, allow_null=True)
```

Comment:
I don't think these are case sensitive, are they?

Reply:
Yes they are

Reply:
I followed the pay request that they sent 

{
  "CLI": "0837867169",
  "Name": "ProdTest",
  "Language": "Snow",
  "Province": "KZN",
  "Gender": "Male",
  "TB_Risk": "High",
  "Responded": "No",
  "TB_Tested": "No",
  "TB_Test_Results": "Yes",
  "Screen_timeStamp": "2023-04-15 10:17:23"
}